tools/preseal/preseal.c

```diff
@@ -0,0 +1,195 @@
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/settings.h>
```

Comment:
Does this tool really work against a physically different TPM? Or does it need to be run using the same TPM??

Reply:
It needs to run on the same TPM since it needs to seal the public key to NV